src/MagentoHackathon/Composer/Magerun/DeployCommand.php

```diff
@@ -8,6 +8,11 @@
 
```

Comment:
Hi @anzin 
Could we drop this file ?

Reply:
Hello @xmav, I didn't find using this command and deleted this file, thank you for review.